scripts/files/fs_s3.py

```diff
@@ -56,17 +59,18 @@ def read(path: str, needs_credentials: bool = False) -> bytes:
 
         s3_object = s3.Object(s3_path.bucket, key)
         file: bytes = s3_object.get()["Body"].read()
+    except s3.meta.client.exceptions.NoSuchBucket as nsb:
+        get_log().error("s3_bucket_not_found", path=path, error=f"The specified bucket does not seem to exist: {nsb}")
```

Comment:
nit: I see this is just copied but, could this be "The specified bucket does not exist"?